implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -342,6 +342,9 @@ impl NodeManager {
             (Post, ["node", "secure_channel"]) => {
                 self.create_secure_channel(req, dec).await?.to_vec()?
             }
+            (Post, ["node", "delete_secure_channel"]) => {
+                self.delete_secure_channel(req, dec).await?.to_vec()?
+            }
```

Comment:
```suggestion
            (Delete, ["node", "secure_channel"]) => {
                self.delete_secure_channel(req, dec).await?.to_vec()?
            }
```

We're trying to model an HTTP like interface here, so it would better if this is Delete

Reply:
Oops, sure.